eclass/distutils-r1.eclass

```diff
@@ -1442,12 +1442,20 @@ distutils-r1_python_compile() {
 			fi
 			;;
 		maturin)
-			# auditwheel may attempt to auto-bundle libraries, bug #831171
-			local -x MATURIN_PEP517_ARGS=--skip-auditwheel
-
-			# support cargo.eclass' IUSE=debug if available
-			in_iuse debug && use debug &&
-				MATURIN_PEP517_ARGS+=" --cargo-extra-args=--profile=dev"
+			if has_version '>=dev-util/maturin-0.13'; then
+				# auditwheel may auto-bundle libraries (bug #831171),
+				# also support cargo.eclass' IUSE=debug if available
+				local -x MATURIN_PEP517_ARGS="
+					--jobs=$(makeopts_jobs)
+					--skip-auditwheel
+					$(in_iuse debug && usex debug --profile=dev '')
+				"
+			else
+				# legacy support, can cleanup when depend on >=0.13
+				local -x MATURIN_PEP517_ARGS=--skip-auditwheel
+				in_iuse debug && use debug &&
+					MATURIN_PEP517_ARGS+=" --cargo-extra-args=--profile=dev"
```

Comment:
I suppose you could use the `$()` style here too for consistency/reaadability.

Reply:
Sure I guess, should be short lived but made it the same as above.